en/asgardeo/docs/apis/restapis/functionality.yaml

```diff
@@ -0,0 +1,346 @@
+openapi: 3.0.0
+info:
+  description: This is the RESTful API for handling user functionality management in Asgardeo. 
+  version: "1.0.0"
+  title: User Functionality Management API
```

Comment:
This API is not properly used in IS as well, better not to promote in Asgardeo

Reply:
Let's decide on https://github.com/wso2-enterprise/asgardeo-product/issues/23774

Reply:
https://github.com/wso2/docs-is/pull/4563/commits/b301a4c53580a21095f73e3bc7d80a7022217888